src/VisualStudio/Core/Def/LanguageService/AbstractPackage.cs

```diff
@@ -27,18 +27,13 @@ internal IComponentModel ComponentModel
 
     protected override async Task InitializeAsync(CancellationToken cancellationToken, IProgress<ServiceProgressData> progress)
     {
-        await base.InitializeAsync(cancellationToken, progress).ConfigureAwait(true);
```

Comment:
>InitializeAsync [](http://example.com/codeflow?start=19&length=15)

Base class documents that calling it's InitializeAsync and OnAfterPackageLoadedAsync methods is not necessary.